tools/circle-upload-codecov.sh

```diff
@@ -2,11 +2,15 @@
 
 set -eo pipefail
 
+PLATFORM=`uname -m`
+[ $PLATFORM == "x86_64" ] && PLATFORM=linux
+BASE_URL="https://uploader.codecov.io/latest/$PLATFORM"
+
 # Source: https://docs.codecov.com/docs/codecov-uploader#integrity-checking-the-uploader
 curl https://keybase.io/codecovsecurity/pgp_keys.asc | gpg --no-default-keyring --keyring trustedkeys.gpg --import
-curl -Os https://uploader.codecov.io/latest/linux/codecov
-curl -Os https://uploader.codecov.io/latest/linux/codecov.SHA256SUM
-curl -Os https://uploader.codecov.io/latest/linux/codecov.SHA256SUM.sig
+curl -Os "$BASE_URL/codecov"
```

Comment:
not critical, but when using parameter expansion inside the string, it's better to use the full form ```"some text${variable}some more text"```